### PR TITLE
remove demo refrences

### PR DESCRIPTION
**Pull request by EyalIlan**
remove demo refrences in utils.ts env. widget.data.servie.ts

### CODE REVIEW

**Review by yuvalbl — APPROVED** [review]

File: src/services/news.data.service.ts [hunk 1]
```diff
@@ -43,7 +41,7 @@ export function fetchNews(source = '', offSet = 0, limit = 100): Promise<any> {
       .get(url)
       .then((res) => res.data)
       // if showDemoCards - add demoNewsflash to newsflash data
```
Comment (danielsh28) [inline src/services/news.data.service.ts:43]: remove this comment

File: src/services/news.data.service.ts [hunk 2]
```diff
@@ -43,7 +41,7 @@ export function fetchNews(source = '', offSet = 0, limit = 100): Promise<any> {
...
-      .then((data) => (showDemoCards ? addDemoNewsflash(data) : data))
+      .then((data) => (data))
```
Comment (danielsh28) [inline src/services/news.data.service.ts:44]: don't think we need `then((data) => (data))`

File: src/services/widgets.data.service.ts [hunk 3]
```diff
@@ -39,9 +38,7 @@ export const fetchWidgets = async ({
   yearAgo,
   gpsId,
 }: IWidgetInput): Promise<ILocationData | undefined> => {
-  if (showDemoCards && newsId === DEMO_ID) {
-    return getDemoWidgetData();
-  }
+
```
Comment (danielsh28) [inline src/services/widgets.data.service.ts:41]: remove empty line

File: src/utils/utils.ts [hunk 4]
```diff
@@ -46,8 +46,7 @@ AUTH_LOGIN_GOOGLE_URL.searchParams.append('redirect_url', `${window.location.ori
 export const mapApiKey = process.env.REACT_APP_GOOGLE_MAP_KEY ? process.env.REACT_APP_GOOGLE_MAP_KEY : '';
 
 export const showDemoCards = process.env.REACT_APP_SHOW_DEMO_CARDS === 'true';
-// demo id as it wil be appeared in url route
-export const DEMO_ID = 999;
+
```
Comment (danielsh28) [inline src/utils/utils.ts:49]: remove empty line

**Review by danielsh28 — APPROVED** [review]

**Pull request merged**